Added Overview of RHOBS, Observatorium, Telemeter and MST (#21)
rhobs/handbook: content/Projects/Observability/RHOBS/mst.md

# RHOBS: MST

> For RHOBS Overview see [this document](README.md)

TBD

## Support

TBD

## Service Level Agreement

![SLO](../../../assets/slo-def.png)

> If you manage [“Observatorium”](../observatorium.md) the Service Level Objectives can go ultra high in all dimensions like availability, and data loss. The freshness aspect for read APIs is trickier as among other things it also depends on client collection pipeline availability which is out of Observatorium scope.

Reviewer: ```suggestion
If you manage [“Observatorium”](../observatorium.md) the Service Level Objectives can go ultra high in all dimensions like availability, and data loss. The freshness aspect for read APIs is trickier as it also depends on client collection pipeline availability, among other things, which is out of the scope of Observatorium.
```

> If you manage [“Observatorium”](../observatorium.md) the Service Level Objectives can go ultra high in all dimensions like availability, and data loss. The freshness aspect for read APIs is trickier as among other things it also depends on client collection pipeline availability which is out of Observatorium scope.

Reviewer 2: ```suggestion
If you manage [“Observatorium”](../observatorium.md), the Service Level Objectives can go ultra high in all dimensions such as availability and data loss. The freshness aspect for read APIs is trickier as, among other things, it also depends on client collection pipeline availability which is out of the Observatorium scope.
```

> RHOBS as Observatorium as a Service has currently established following, default, Service Level Objectives. This is based on infrastructure dependencies we have listed [here (intenral)](https://visual-app-interface.devshift.net/services#/services/rhobs/app.yml).

Reviewer: ```suggestion
RHOBS as Observatorium as a Service has currently established the following, default, Service Level Objectives. This is based on the infrastructure dependencies we have listed [here (internal)](https://visual-app-interface.devshift.net/services#/services/rhobs/app.yml).
```

Reviewer: What do we mean by "RHOBS as Observatorium as a service"?

Reviewer 2: Also drop the commas around "default."

> > NOTE: We envision more granular tiers of SLO in future for different tenants, for better QoS levels. Currently, all tenants has the same SLOs.

Reviewer: ```suggestion
> NOTE: We envision more granular tiers of SLO in future for different tenants, for better QoS levels. Currently, all tenants have the same SLOs.
```

> > NOTE: We envision more granular tiers of SLO in future for different tenants, for better QoS levels. Currently, all tenants has the same SLOs.

Reviewer 2: ```suggestion
> NOTE: We envision future improvements to quality of service by offering additional SLO tiers for different tenants. Currently all tenants have the same SLOs.
```
